pkg/util/helper/workstatus.go

```diff
@@ -283,6 +283,10 @@ func worksFullyApplied(aggregatedStatuses []workv1alpha2.AggregatedStatusItem, t
 			return false
 		}
 
+		if aggregatedStatusItem.Health != workv1alpha2.ResourceHealthy {
```

Comment:
If we have 2 member clusters, and it's healthy for one and not the other. Should be it considered fully applied? or not applied?